### ISSUE TITLE
The Genesis File for setting up a local test network has changed

### ISSUE THREAD

**Issue by anthonyjp87** [ethereum/homestead-guide#410]
Hi, 

There is an issue with the Genesis file as described in the homestead-guide/source/network/test-networks.rst. I believe the format has changed.

currently in homestead doc: 
{
    "nonce": "0x0000000000000042",     "timestamp": "0x0",
    "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "extraData": "0x0",     "gasLimit": "0x8000000",     "difficulty": "0x400",
    "mixhash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "coinbase": "0x3333333333333333333333333333333333333333",     "alloc": {     }
}

Currently supported in Ethereum based on my testing (12/22/17) and described at this link https://github.com/ethereum/go-ethereum/wiki/Private-network
{
    "config": {
        "chainId": 15,
        "homesteadBlock": 0,
        "eip155Block": 0,
        "eip158Block": 0
    },
    "difficulty": "200000000",
    "gasLimit": "2100000",
    "alloc": {
        "7df9a875a174b3bc565e6424a0050ebc1b2d1d82": { "balance": "300000" },
        "f41c74c9ae680c1aa78f42e5647a62f353b7bdde": { "balance": "400000" }
    }
}

**Comment by paulvi** [comment]
Yes, the source `/source/network/test-networks.rst` that is also published at http://ethdocs.org/en/latest/network/test-networks.html
is outdated

There is question if this repository is cared for at all #412
